app/views/about-your-organisation/index.html

```diff
@@ -38,81 +94,149 @@ <h1 class="govuk-heading-xl">
-            {{ data["town-or-city"] }}<br />
-          {% endif %}
-          {% if data["county"] %}
-            {{ data["county"] }}<br />
-          {% endif %}
-          {% if data["postcode"] %}
-            {{ data["postcode"] }}<br />
-          {% endif %}
-        {% else %}
-          Empty
-        {% endif %}
-      {% endset %}
-
+      <h3 class="govuk-heading-m">Contact details</h3>
       {{ govukSummaryList({
         rows: [{
+          classes: macros.requiredSummaryRowClasses("email"),
           key: {
             text: "Email address"
           },
           value: {
-            text: value("email") or "Empty",
-            classes: "app-summary-list__value--empty" if not value("email")
+            text: value("email") or "Empty"
+          },
+          actions: {
+            items: [{
+              href: "/about-your-organisation/contact",
+              text: "Change",
+              visuallyHiddenText: "email address"
+            }]
           }
         }, {
+          classes: macros.requiredSummaryRowClasses("telephone"),
           key: {
             text: "Telephone number"
           },
           value: {
-            text: value("telephone") or "Empty",
-            classes: "app-summary-list__value--empty" if not value("telephone")
+            text: value("telephone") or "Empty"
+          },
+          actions: {
+            items: [{
+              href: "/about-your-organisation/contact",
+              text: "Change",
+              visuallyHiddenText: "telephone number"
+            }]
           }
         }, {
+          classes: macros.requiredSummaryRowClasses("website"),
           key: {
             text: "Website"
           },
           value: {
-            text: value("website") or "Empty",
-            classes: "app-summary-list__value--empty" if not value("website")
+            text: value("website") or "Empty"
+          },
+          actions: {
+            items: [{
+              href: "/about-your-organisation/contact",
+              text: "Change",
+              visuallyHiddenText: "website"
+            }]
           }
         }, {
+          classes: macros.requiredSummaryRowClasses("building-and-street"),
           key: {
             text: "Contact address"
           },
           value: {
-            html: contactAddressHtml,
-            classes: "app-summary-list__value--empty" if not value("building-and-street")
+            html: contactAddressHtml | nl2br
+          },
+          actions: {
+            items: [{
+              href: "/about-your-organisation/contact#contact-address",
+              text: "Change",
+              visuallyHiddenText: "contact address"
+            }]
           }
         }]
       }) }}
 
-      <h3 class="govuk-heading-m govuk-!-font-size-27">
-        <a href="/about-your-organisation/about">About your organisation</a>
-      </h3>
+      <h3 class="govuk-heading-m">About your organisation</h3>
```

Comment:
Feels weird to have an `h3` which matches the `h1`. And where did `h2` go? 😄  (I know you've not changed this)

Reply:
😭 (Have updated the heading levels)